tools/build.csx

```diff
@@ -3,8 +3,6 @@
 #r "System.Net.Http"
 #r "System.Xml.Linq"
 
-using System.Net.Http;

```

Comment:
Maybe remove the `#r`s too?

Reply:
Oh, and don't worry about fixups. I can keep up with the changes if you just amend.

Reply:
> Maybe remove the #rs too?

Oops. I should have thought of it